scripts/buildCSS.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const CleanCSS = require('clean-css');
+const compileCSS = require('react-with-styles-interface-css-compiler');
+const registerMaxSpecificity = require('react-with-styles-interface-css/dist/utils/registerMaxSpecificity').default;
+const registerCSSInterfaceWithDefaultTheme = require('./utils/registerCSSInterfaceWithDefaultTheme').default;
+
+const args = process.argv.slice(2);
+const optimizeForProduction = args.includes('-o') || args.includes('--optimize');
+
+registerMaxSpecificity(0);
+registerCSSInterfaceWithDefaultTheme();
+
+const format = new CleanCSS({
+  level: optimizeForProduction ? 2 : 0,
+  format: 'beautify',
+  inline: ['none'],
+});
+const CSS = compileCSS('./scripts/renderAllComponents.jsx');
+const formattedCSS = `${format.minify(CSS).styles} \n`;
+const dir = optimizeForProduction ? './lib/css' : './css';
+
+if (!fs.existsSync(dir)) {
+  fs.mkdirSync(dir);
+}
+
+const outputFilePath = optimizeForProduction ? `${dir}/_styles.css` : `${dir}/styles.css`;
```

Comment:
the underscore is *probably* not necessary (it was there for some backwards compatibility w/ react-dates). It might be nice to name this file `rheostat.css` so that it is easy to understand in the context of a bigger app.